backend/server/index.js

```diff
@@ -183,7 +215,7 @@ app.get('/profile/:name/busker', async ({ params }, res) => {
   res.send('TEST');
```

Comment:
This causes an error. I deleted it on my end and it seems to work fine, maybe leftover from when you were trying stuff out.